.devcontainer/Dockerfile

```diff
@@ -1,86 +1,89 @@
 FROM python:3.6
-LABEL maintainer “Shiho ASA<asashiho@mail.asa.yokohama>”
+LABEL maintainer "Shiho ASA<asashiho@mail.asa.yokohama>"
+
+# definitely the most annoying part is getting credentials
+COPY ./credentials /opt/aws
```

Comment:
Baking in credentials to the image is very bad practice. That will never pass any security review.

Reply:
I can change it so a script copies into the container rather than the image. I'd set it up this way first to avoid cross-platform issues but now I can use Linux Docker.

Reply:
That sounds good ...